_data/authors.yaml

```diff
@@ -322,4 +322,9 @@ galderz:
 mjurc:
   name: "Michal Jurč"
   email: "mjurc@redhat.com"
-  job_title: "Senior Software Quality Engineer"
\ No newline at end of file
+  job_title: "Senior Software Quality Engineer"
+bennetelli:
+  name: "Bennet Schulz"
+  email: "bschulz@redhat.com"
```

Comment:
Add the email hash too. You can generate it here: https://daveceddia.com/calculate-gravatar-hash/